plugin/evm/orderbook/order_types.go

```diff
@@ -16,8 +16,8 @@ type ContractOrder interface {
 	Map() map[string]interface{}
 }
 
-// LimitOrder type is copy of Order struct defined in LimitOrderbook contract
-type LimitOrder struct {
+// OrderCommon is the set of common fields among the order types
+type OrderCommon struct {
```

Comment:
please suggest a better name for this struct

Reply:
maybe `BaseOrder` 😛